app/components/finance/npq/payment_overviews/payment_overview_table.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    module PaymentOverviews
+      class PaymentOverviewTable < BaseComponent
+        include FinanceHelper
+
+        def initialize(contract, statement, lead_provider)
+          @contract = contract
+          @statement = statement
+          @lead_provider = lead_provider
+        end
+
+      private
+
+        attr_accessor :statement, :contract, :lead_provider
+
+        def service_fees
+          @service_fees ||= PaymentCalculator::NPQ::ServiceFees.call(contract: contract)
+        end
+
+        def output_payments
+          @output_payments ||= PaymentCalculator::NPQ::OutputPayment.call(
+            contract: contract,
+            total_participants: statement.participant_declarations.unique_paid_payable_or_eligible.count,
```

Comment:
fyi you will lose the feature of being able to view the "current" statement with line. as the "current" statement does not at the moment have any declarations assigned to it. so this will make the current view return zero. Unless `statement` here is some new "anonymous statement" which i don't believe it to be

Reply:
Have updated this now